doc/release_notes/release_notes_3.0.rst

```diff
@@ -50,7 +69,14 @@ that is essential to build ACRN. Compared to previous versions, ACRN v3.0 adds
 the following hardware information to board XMLs to support new features and
 fixes.
 
-  - TBD
+  - Add a cmdline option to inject LLC CAT capability
+  - Collect all information about SR-IOV devices.
+  - Extract all serial ttys and virtio input device in the native environment.
+  - Extract common ioapic information such as ioapic id, address, gsi base and gsi num.
+  - Add another level of `die` node even though the hardware reports die topology in CPUID.
```

Comment:
I'm not sure what this means exactly, can you explain some more?

Reply:
updated.